testFiles/Link-U/README.md

```diff
@@ -0,0 +1,232 @@
+# AVIF Example files from [Link-U](https://www.link-u.co.jp/)
+
+ - All files do not contain Exif metadata.
+ - All files are tagged as MIAF compatible.
+ - All files are tagged as compatible with the AVIF Baseline or Advanced Profile if possible.
+ - All images have the "reduced_still_picture_header" and "still_picture" flags set to 1 in the AV1 Sequence Header.
+ - All images are licensed under [CC-BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/deed.en).
```

Comment:
I understand that you produced the AVIF files and provide them with a CC-BY-SA license, but I see that you retrieved the images from some web site. Can you confirm also that the source images have the same license?

Reply:
Hello, I am an author of original images. I added the license badges to pages. Please check.

https://hexe.net/2017/11/27/12:27:02/
https://hexe.net/2018/12/24/18:59:01/
https://hexe.net/2017/12/02/16:33:53/

Reply:
I contacted the authors and asked them to specify license.